packages/integration-karma/test/component/dynamic-imports/x/dynamic/dynamic.js

```diff
@@ -0,0 +1,13 @@
+import { LightningElement, track } from 'lwc';
+export default class DynamicCtor extends LightningElement {
+    @track customCtor;
+
+    connectedCallback() {
+        this.loadCtor();
+    }
+
+    async loadCtor() {
+        const ctor = await import('x-ctor');
```

Comment:
this worries me. in theory this should be equivalent to the module specifier used for static imports, but it is not:

```
// static import
import StaticCtor from 'x/ctor';
// dynamic
const DyanmicCtor = await import('x-ctor');
```

Am I missing something?

Reply:
oh, I think this is because `registerForLoad('x-ctor', DynamicCtor);`, in which case we are fine, but maybe just update the name to match.

Reply:
yeah I will change it, its jus they way I did the load dance.